deploy/namespace-operator.yaml

```diff
@@ -187,7 +187,7 @@ spec:
               ephemeral-storage: "50Mi"
       containers:
         - name: seccomp-operator
-          image: k8s.gcr.io/seccomp-operator/seccomp-operator:v0.1.0
+          image: k8s.gcr.io/seccomp-operator/seccomp-operator:0.1.0
```

Comment:
Don't we tag k8s releases with a `v` prefix?

Reply:
Yes the tag will contain the prefix, but I prefer having the container image without it. I think it's more common. WDYT?

Reply:
Ah no I see we do it the same way in Kubernetes, which should be fine.